### ISSUE TITLE
Commands fail on latest version of conda (4.5.2) with ImportError

### ISSUE THREAD

**Issue by hudak** [conda-tools/conda-execute#50]
Breaking changes in conda are causing failures in conda_interface.py

```
(base) root@a0bc50c741e3:/# conda --version
conda 4.5.2
(base) root@a0bc50c741e3:/# conda list conda-execute
# packages in environment at /opt/conda:
#
# Name                    Version                   Build  Channel
conda-execute             0.8.2                      py_0    conda-forge
(base) root@a0bc50c741e3:/# conda execute --help
Traceback (most recent call last):
  File "/opt/conda/bin/conda-execute", line 6, in <module>
    from conda_execute.execute import main
  File "/opt/conda/lib/python2.7/site-packages/conda_execute/execute.py", line 18, in <module>
    import conda_execute.config
  File "/opt/conda/lib/python2.7/site-packages/conda_execute/config.py", line 4, in <module>
    from conda_execute.conda_interface import envs_dirs, pkgs_dirs, rc
  File "/opt/conda/lib/python2.7/site-packages/conda_execute/conda_interface.py", line 17, in <module>
    from conda.config import user_rc_path, sys_rc_path
ImportError: No module named config
(base) root@a0bc50c741e3:/# conda tmpenv list
Traceback (most recent call last):
  File "/opt/conda/bin/conda-tmpenv", line 6, in <module>
    from conda_execute.tmpenv import main
  File "/opt/conda/lib/python2.7/site-packages/conda_execute/tmpenv.py", line 15, in <module>
    from conda_execute.conda_interface import CONDA_VERSION_MAJOR_MINOR
  File "/opt/conda/lib/python2.7/site-packages/conda_execute/conda_interface.py", line 17, in <module>
    from conda.config import user_rc_path, sys_rc_path
ImportError: No module named config
```

**Comment by cvelascof** [comment]
These changes solve this issue for latest conda versions.

```
[user@server ~]$ conda --version
conda 4.5.3
[user@server ~]$ conda list conda-execute
# packages in environment at /home/$USER/miniconda3:
#
# Name                    Version                   Build  Channel
conda-execute             0.8.2                      py_0    conda-forge
[user@server ~]$ conda execute --help
usage: conda-execute [-h] [--force-env] [--verbose | --quiet]
                     [--code [CODE [CODE ...]]]
                     [path] ...

Execute a script in a temporary conda environment.

positional arguments:
  path                  The script to execute.
  remaining_args        Remaining arguments are passed through to the called
                        script.

optional arguments:
  -h, --help            show this help message and exit
  --force-env, -f       Force re-creation of the environment, even if it
                        already exists.
  --verbose, -v         Turn on verbose output.
  --quiet, -q           Prevent any output, other than that of the script
                        being executed.
  --code [CODE [CODE ...]], -c [CODE [CODE ...]]
                        The code to execute.
[user@server ~]$ conda tmpenv list
12 processes (newest created 0:01:30.158292 ago) using /home/$USER/miniconda3/tmp_envs/ce4ca9dbe3080ea75e65 (running PIDs 28339, 28333, 28335, 28338, 28331, 28336, 28304, 28332, 28321, 28340, 28334, 28337)


```

**Comment by pelson** [comment]
Merged in #52.
